src/test/java/org/pesmypetcare/webservice/controller/appmanager/GoogleCalendarRestControllerTest.java

```diff
@@ -0,0 +1,140 @@
+
+/**
+ * @author Marc Simó
+ */
+@SpringBootTest
+@AutoConfigureMockMvc
+@ExtendWith(MockitoExtension.class)
+public class GoogleCalendarRestControllerTest {
+    private static final String TOKEN = "token";
+    private static String jsonEventEntity;
+    private static String jsonEventId;
+    private static List<Event> eventList;
+    private static EventEntity eventEntity;
+    private static String accessToken;
+    private static String owner;
+    private static String petName;
+    private static String urlBase;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private GoogleCalendarService service;
+
+    @BeforeAll
+    public static void setUp() {
+        owner = "My owner";
+        petName = "My petName";
+        accessToken = "tokem";
+        eventList = new ArrayList<>();
+        eventEntity = new EventEntity();
+        urlBase = "/calendar";
+        jsonEventEntity = "{\n"
+            + "  \"id\": \"idtest\",\n"
+            + "  \"summary\":\"Event for tests\",\n"
+            + "  \"location\": \"Wherever you wanna be\",\n"
+            + "  \"description\": \"This is an event I created just to do tests\",\n"
+            + "  \"color\": \"11\",\n"
+            + "  \"emailReminderMinutes\": 10,\n"
+            + "  \"repetitionInterval\": 1,\n"
+            + "  \"startDate\": \"2020-04-26T18:20:38\",\n"
+            + "  \"endDate\": \"2020-04-26T23:20:38\"\n"
+            + "}";
+        jsonEventId = "{\n"
+            + "  \"eventId\": \"idtest\"\n"
+            + "}";
+    }
+
+    @Test
+    public void createSecondaryCalendarShouldReturnStatusOk() throws Exception {
+        willDoNothing().given(service).createSecondaryCalendar(anyString(), anyString(), anyString());
+        mockMvc.perform(post(urlBase + "/" + owner + "/" + petName)
+            .header(TOKEN, accessToken))
+            .andExpect(status().isOk());
+    }
+
+    @Test
+    public void deleteSecondaryCalendarShouldReturnStatusOk() throws Exception {
+        willDoNothing().given(service).deleteSecondaryCalendar(anyString(), anyString(), anyString());
+        mockMvc.perform(delete(urlBase + "/" + owner + "/" + petName)
+            .header(TOKEN, accessToken))
+            .andExpect(status().isOk());
+    }
+
+    @Test
+    public void getAllEventsFromCalendarShouldReturnAnEventListAndStatusOk() throws Exception {
+        willReturn(eventList).given(service).getAllEventsFromCalendar(anyString(), anyString(), anyString());
+        mockMvc.perform(get(urlBase + "/" + owner + "/" + petName)
+            .header(TOKEN, accessToken))
+            .andExpect(status().isOk());
+    }
+
+    @Test
+    public void createEventShouldReturnStatusOk() throws Exception {
+        willDoNothing().given(service).createEvent(anyString(), anyString(), anyString(), isA(EventEntity.class));
+        mockMvc.perform(post(urlBase + "/event/" + owner + "/" + petName)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/event/" appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5403404)